usr/themes/default/sidebar.php

```diff
@@ -55,7 +55,7 @@
                 <?php endif; ?>
                 <li><a href="<?php $this->options->feedUrl(); ?>"><?php _e('文章 RSS'); ?></a></li>
                 <li><a href="<?php $this->options->commentsFeedUrl(); ?>"><?php _e('评论 RSS'); ?></a></li>
-                <li><a href="http://www.typecho.org">Typecho</a></li>
+                <li><a href="https://www.typecho.org">Typecho</a></li>
```

Comment:
```suggestion
                <li><a href="https://typecho.org">Typecho</a></li>
```

@joyqi typecho网站的重定向有些问题，`https://www.typecho.org` 会重定向到 http://typecho.org

Reply:
> @joyqi typecho网站的重定向有些问题，`https://www.typecho.org` 会重定向到 http://typecho.org

已修复